test/fixtures/comprehensions/await/actual.js

```diff
@@ -1,4 +1,4 @@
 f() -/>
-  <- [for x of arr:
+  <- [for now x of arr:
```

Comment:
(few other places like this too... if the compilation was just wrong before, I guess it's fine to leave... )